packages/icsi-lib/src/helpers.ts

```
@@ -0,0 +1,244 @@
+    string,
+    { tokenType: TokenType; tokenName: string; amount: bigint }
+  >();
+
+  for (const tx of transactions) {
+    const tokenKey = JSON.stringify(tx.token_type);
+
+    // Only count non-swept transactions
+    if (tx.sweep_status && 'NotSwept' in tx.sweep_status) {
+      let amount = BigInt(0);
+
+      if (tx.operation && tx.operation[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)